client/src/components/CustomFields.js

```diff
@@ -710,6 +756,7 @@ CustomFieldsContainer.propTypes = {
   fieldsConfig: PropTypes.object,
   formikProps: PropTypes.object,
   parentFieldName: PropTypes.string.isRequired,
+  linkToComp: PropTypes.func.isRequired,
```

Comment:
It is here because of circular dependency, there were 2 options, 

1- Move readonly components to a different file so only modify ReadonlyCustomFields props
2- Keep everything in the same file and change all LinkTo components to prop.